packages/data-broker/src/services/kobo/KoBoTranslator.js

```diff
@@ -31,17 +38,16 @@ export class KoBoTranslator {
     const dataValues = {};
     for (const [tupaia, { koboQuestionCode, answerMap }] of Object.entries(questionMapping)) {
       if (restOfFields[koboQuestionCode] !== undefined) {
-        let koboValue = restOfFields[koboQuestionCode];
-        if (answerMap) {
-          koboValue = answerMap[koboValue] || koboValue;
-        }
-        dataValues[tupaia] = koboValue;
+        const koboValue = restOfFields[koboQuestionCode];
+        dataValues[tupaia] =
+          answerMap?.[koboValue] !== undefined ? answerMap[koboValue] : koboValue;
       }
     }
 
     return {
       event,
       eventDate,
+      koboUser,
```

Comment:
```suggestion
      assessor: koboUser,
```
Might as well use a generic term that we might reuse across sync services